bridges/otelzap/core_test.go

```diff
@@ -8,12 +8,30 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
 
+	"go.opentelemetry.io/otel/log"
 	"go.opentelemetry.io/otel/log/global"
 	"go.opentelemetry.io/otel/log/logtest"
 	"go.opentelemetry.io/otel/sdk/instrumentation"
 )
 
+var testMessage = "log message"
+
+func TestCore(t *testing.T) {
```

Comment:
This doesn't need to be a package variable.

```suggestion
func TestCore(t *testing.T) {
	testMessage := "log message"
```

Reply:
we may need this for other tests as well, so

Reply:
It's just a string. We also can move it when we need it.